stdlib/public/core/UnsafeRawBufferPointer.swift.gyb

```diff
@@ -348,6 +348,56 @@ public struct Unsafe${Mutable}RawBufferPointer
     return 0
   }
 
+  %  if mutable:
+    /// Initializes memory in the buffer with the elements of
+    /// `source` and binds the initialized memory to type `T`.
+    ///
+    /// Returns an iterator to any elements of `source` that didn't fit in the 
+    /// buffer, and an index to the point in the buffer one past the last element
+    /// written.
+    ///
+    /// - Precondition: The memory in `startIndex..<(source.count *
+    ///   MemoryLayout<T>.stride)` is uninitialized or initialized to a
+    ///   trivial type. The buffer must contain sufficient memory to
+    ///   accommodate at least `source.underestimateCount` elements.
+    ///
+    /// - Postcondition: The memory at `self[startIndex..<returned index] *
+    ///   MemoryLayout<T>.stride` is bound to type `T`.
+    ///
+    /// - Postcondition: The `T` values at `self..<self + source.count *
+    ///   MemoryLayout<T>.stride` are initialized.
```

Comment:
This postcondition isn't accurate any more, since `count` doesn't on a sequence and the buffer only initializes items until it runs out of room—hard to figure out how to write this concisely.